src/app/ubs/ubs/components/ubs-personal-information/ubs-order-address/ubs-order-address.component.ts

```diff
@@ -31,21 +33,25 @@ export class UbsOrderAddressComponent implements OnInit, OnDestroy {
   maxAddressLength = 4;
   $isAddressLoading = this.store.pipe(select(isAddressLoadingSelector));
   private $destroy: Subject<void> = new Subject<void>();
+  private addressData: CAddressData;
 
   constructor(
     private route: ActivatedRoute,
     private dialog: MatDialog,
     private addressValidator: AddressValidator,
-    private store: Store
-  ) {}
+    private store: Store,
```

Comment:
Members `store: Store`, `addressData` and  `languageService: LanguageService` is never reassigned; mark it as `readonly`.